mapsims/runner.py

```diff
@@ -17,6 +18,7 @@
 from .utils import DEFAULT_INSTRUMENT_PARAMETERS
 
 try:
+    if os.environ['DISABLE_MPI']: raise ImportError
```

Comment:
@msyriac you need to use `os.environ.get`

Reply:
Fixed